scripts/install-build-deps

```diff
@@ -145,7 +145,21 @@ SEAGATE_GITHUB_CORTX_DEPS_REPO=https://github.com/Seagate/cortx/releases/downloa
 if ! $use_ansible ; then
   case $(distro_type) in
       redhat)
-        yum-config-manager --enable powertools
+        source /etc/os-release
+        if [[ $NAME == "Rocky Linux" ]];
+        then
+	    if [[ ! `yum repolist | grep Power` ]]
+            then
```

Comment:
Style: use `if ...; then` on the same line.

Reply:
Done.